0-bootstrap/main.tf

```diff
@@ -55,6 +55,15 @@ module "seed_bootstrap" {
   sa_enable_impersonation = true
   parent_folder           = var.parent_folder == "" ? "" : local.parent
   skip_gcloud_download    = var.skip_gcloud_download
+  project_labels = {
+    environment       = "bootstrap"
+    application_name  = "seed-bootstrap"
+    billing_code      = "1234"
+    primary_contact   = "example1"
+    secondary_contact = "example2"
+    business_code     = "bu1"
```

Comment:
should we  use `abcd` since this project is not tied to an actual business unit ?